website/src/pages/index.js

```diff
@@ -0,0 +1,46 @@
+import clsx from 'clsx';
+import Link from '@docusaurus/Link';
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+import Layout from '@theme/Layout';
+import HomepageFeatures from '@site/src/components/Homepage/HomepageFeatures';
+
+import Heading from '@theme/Heading';
+import styles from './index.module.scss';
+
+function HomepageHeader() {
+  const {siteConfig} = useDocusaurusContext();
+  return (
+    <header className={styles.heroBanner}>
+      <div className="container">
+        <h1 className={styles.title}>{siteConfig.title}</h1>
+        <h2 className={styles.tagline}>{siteConfig.tagline}</h2>
+        <div className={styles.buttons}>
+          <Link
+            className="button button--secondary button--lg"
+            to="/docs/guides/integrating-with-testing-frameworks">
+            Getting started with Copilot
+          </Link>
+          {/* Add an img for the gif */}
+          <img
+            src="/img/homepage/copilot-demo.gif"
```

Comment:
This is cursed way to reference assets — it won't work on addresses like `https://wix-incubator.github.io/detox-copilot/` where you have some prior base path such as `/detox-copilot`. 

https://docusaurus.io/docs/markdown-features/assets#images

Read the docs, please. You have either `require(...).default` or `import` option in such case.